README.md

```diff
@@ -0,0 +1,62 @@
+A client library for authenticating with the ISIC Archive from a SPA (single page application).
+
+![npm](https://img.shields.io/npm/v/@isic/client)
+
+
+## Description
+
+The ISIC client provides support for authenticating with the ISIC Archive using the Auth Code with PKCE flow of OAuth2.0.
+
+## Usage
+
+```bash
+$ yarn add @isic/client
+```
+
+```js
+// create an isic client
+const client = new IsicClient(
+  'v1odYySCetBht6DT9svQdAkvmVXrRHOwIIGNk6JG',
```

Comment:
Not sure if we want to encourage people to hardcode this value. Ideally, it should be coming from something like `process.env.CLIENT_ENV`. Maybe provide the dev client ID in another comment, rather than hardcoded into the example.